addons/sale_management/models/sale_order.py

```diff
@@ -208,6 +207,26 @@ class SaleOrderOption(models.Model):
     quantity = fields.Float('Quantity', required=True, digits='Product Unit of Measure', default=1)
     sequence = fields.Integer('Sequence', help="Gives the sequence order when displaying a list of optional products.")
 
+    def _update_price_discount(self):
```

Comment:
```suggestion
    def _update_price_and_discount(self):
```

would be more clear imho :)